packages/blocks/src/embed-synced-doc-block/embed-synced-doc-block.ts

```diff
@@ -210,6 +213,35 @@ export class EmbedSyncedDocBlockComponent extends EmbedBlockElement<
     this._selectBlock();
   }
 
+  private _buildPreviewSpec = (name: 'page:preview' | 'edgeless:preview') => {
+    const nextDepth = this.depth + 1;
+    const previewSpecBuilder = SpecProvider.getInstance().getSpec(name);
```

Comment:
What's the definition of `SpecBuilder` and `PreviewSpec`?

Reply:
https://github.com/toeverything/blocksuite/pull/6976